packages/autorest.typescript/src/utils/valueHelpers.ts

```diff
@@ -149,9 +157,11 @@ export function getParameterAssignment(exampleValue: ExampleValue, isRLCSample:
         }
         let propRetValue: string;
         if (propName.indexOf("/") > -1 || propName.match(/^\d/)) {
-          propRetValue = `"${propName}": ` + getParameterAssignment(property);
+          propRetValue =
+            `"${propName}": ` + getParameterAssignment(property, isRLCSample);
```

Comment:
Have you tried to use your branch generate rlc for arm-compute and arm-network ?

Reply:
Yes but we still have issues when compilation.

For arm-network
```
samples-dev/applicationGatewayWafDynamicManifestsDefaultGetSample.ts:26:24 - error TS2322: Type '"2021-05-01"' is not assignable to type '"2022-05-01"'.

26     queryParameters: { "api-version": "2021-05-01" }
                          ~~~~~~~~~~~~~

  src/parameters.ts:465:3
    465   "api-version": "2022-05-01";
          ~~~~~~~~~~~~~
    The expected type comes from property 'api-version' which is declared here on type 'ApplicationGatewayWafDynamicManifestsDefaultGetQueryParamProperties & Record<string, unknown>'

samples-dev/applicationGatewayWafDynamicManifestsGetSample.ts:27:24 - error TS2322: Type '"2021-05-01"' is not assignable to type '"2022-05-01"'.

27     queryParameters: { "api-version": "2021-05-01" }
```